termios/pty_freebsd.go:10:15: undefined: unix

The "o" project https://github.com/xyproto/o fails:
```
 github.com/pkg/term/termios
vendor/github.com/pkg/term/termios/pty_freebsd.go:10:15: undefined: unix
vendor/github.com/pkg/term/termios/pty_freebsd.go:19:22: undefined: unix
vendor/github.com/pkg/term/termios/pty_freebsd.go:24:19: undefined: unix
vendor/github.com/pkg/term/termios/pty_freebsd.go:33:19: undefined: unix
*** Error code 2
```

This is a regression between c3ffed290a03 and v1.1.0 (it was building fine in c3ffed290a03).

FreeBSD 12.2

--- a/term_bsdu.go
+++ b/term_bsdu.go
@@ -2,7 +2,11 @@
 
 package term
 
-import "golang.org/x/sys/unix"
+import (
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
 
 type attr unix.Termios
 

--- a/termios/pty_freebsd.go
+++ b/termios/pty_freebsd.go
@@ -3,6 +3,8 @@ package termios
 import (
 	"fmt"
 	"unsafe"
+
+	"golang.org/x/sys/unix"
 )
 
 func posix_openpt(oflag int) (fd uintptr, err error) {

